app/kumactl/cmd/install/testdata/install-control-plane.dump-values.yaml

```diff
@@ -681,7 +681,9 @@ postgres:
     mode: disable # ENV: KUMA_STORE_POSTGRES_TLS_MODE
     # -- Whether to disable SNI the postgres `sslsni` option.
     disableSSLSNI: false # ENV: KUMA_STORE_POSTGRES_TLS_DISABLE_SSLSNI
-    # -- Secret name that contains the CA tls.crt, tls.key and CA bundle
+    # -- Secret name that contains the root CA bundle
```

Comment:
nit for comment: "CA bundle".
CA bundle - root CA + all intermediate CAs
root CA - first CA in the chain
Also specify what is the file name in this secret

Reply:
changed